lapce-app/src/settings.rs

```diff
@@ -471,7 +471,7 @@ pub fn settings_view(
         }),
         stack((
             container({
-                text_input(search_editor, || false)
+                text_input(search_editor, || true)
```

Comment:
This doesn't need to be true. 

The `request_focus` down will make `keyboard_focus` to be `true`